app/Http/Requests/PostRequest.php

```diff
@@ -21,6 +21,10 @@ public function authorize()
      */
     public function rules()
     {
+        if ($this->method() === 'PATCH') {
+            return $this->patchRules();
```

Comment:
It might be nice to DRY up these rule definitions with `array_only`:

```php
$rules = [
    ...
];

$updatableFields = ['text', 'quantity', 'status'];
if ($this->method() === 'PATCH') {
    return array_only($rules, $updatableFields)
}

return $rules;
```

Reply:
ah yea nice